subsys/net/buf.c

```diff
@@ -289,11 +289,11 @@ struct net_buf *net_buf_alloc_len(struct net_buf_pool *pool, size_t size,
 #if defined(CONFIG_NET_BUF_POOL_USAGE)
 			NET_BUF_WARN("%s():%d: Pool %s blocked for %u secs",
 				     func, line, pool->name,
-				     (k_uptime_get_32() - ref) / MSEC_PER_SEC);
+				     (k_uptime_get_32() - ref) / K_SECONDS(1));
 #else
```

Comment:
The rest of the fixes in this PR seem good, but I'm not sure I agree with this. The way the K_SECONDS, K_MSEC, K_MINUTES, K_HOURS macros work, is that you don't need to know what the "native" representation of time is that's passed to the kernel APIs. It might be milliseconds, but it could be something else that the macros convert the input to. The above change assumes that the macro converts to milliseconds, whereas with MSEC_PER_SEC it's at least explicit.

Reply:
+1 to @jhedberg's comment, I'd also say that division by MSEC_PER_SEC is easier to understand than division by K_SECONDS(1).

Reply:
Ok, lets leave MSEC_PER_SEC there then.